static/css/main.less

```diff
@@ -746,8 +746,13 @@ span.var {
   position:absolute;
   bottom:10px; right:10px;
 }
-.spread_view .hang_inside {
-  right:auto; left:10px;
+.spread_view .sign_view {
+  position: absolute;
+  right:0; bottom:0;
+
```

Comment:
Put `max-width:35%;` and `max-height:100%;` here to prevent layout bleeding/preserve aspect ratio.